### DISCUSSION TITLE
Only route traffic to/from Transmission through VPN

### DISCUSSION THREAD

**Discussion by QfeEBQciZg** [haugene/docker-transmission-openvpn#2486] [Q&A] [answered]
Hi,

I'm new to docker networking. I noticed that all the host network traffic is being routed through the VPN once the container is up. Is there way to only route the traffic to/from Transmission through VPN?

**Comment by pkishino** [comment, marked as answer]
Then you have setup the networking incorrectly. 
please post your config

**Reply by QfeEBQciZg** [reply]
ah, it seems the network mode was initially set to `host`, the routing works correctly after I switched to `bridge`.

Thanks for the clue!
